### ISSUE TITLE
Payload undefined

### ISSUE THREAD

**Issue by gliide** [A21z/node-cd#10]
Nice idea for a module, very useful -

however currently getting 'payload undefined' on the console, and a 403 response on GitHub

**Comment by Insoleet** [comment]
Same problem

**Comment by A21z** [comment]
Could you please try with branch 2.0.0? I'll release it soon but haven't tested this bug on it yet.

**Comment by Insoleet** [comment]
On branch 2.0.0 I get the following error : 

```
qindex-1 TypeError: object is not a function
index-1     at callbacks (/var/www/sakia-wallet.org/node-cd/src/node_modules/express/lib/router/index.js:165:11)
index-1     at param (/var/www/sakia-wallet.org/node-cd/src/node_modules/express/lib/router/index.js:139:11)
index-1     at pass (/var/www/sakia-wallet.org/node-cd/src/node_modules/express/lib/router/index.js:146:5)
index-1     at Router._dispatch (/var/www/sakia-wallet.org/node-cd/src/node_modules/express/lib/router/index.js:173:5)
index-1     at Object.router (/var/www/sakia-wallet.org/node-cd/src/node_modules/express/lib/router/index.js:33:10)
index-1     at next (/var/www/sakia-wallet.org/node-cd/src/node_modules/express/node_modules/connect/lib/proto.js:190:15)
index-1     at Object.static (/var/www/sakia-wallet.org/node-cd/src/node_modules/express/node_modules/connect/lib/middleware/static.js:56:61)
index-1     at next (/var/www/sakia-wallet.org/node-cd/src/node_modules/express/node_modules/connect/lib/proto.js:190:15)
index-1     at Object.urlencodedParser [as handle] (/var/www/sakia-wallet.org/node-cd/node_modules/body-parser/lib/types/urlencoded.js:81:44)
index-1     at next (/var/www/sakia-wallet.org/node-cd/src/node_modules/express/node_modules/connect/lib/proto.js:190:15)
```

**Comment by A21z** [comment]
OK I'll have a look today and try to release 2.0.0

**Comment by Insoleet** [comment]
Please note that the hook is behind a reverse proxy.

**Comment by A21z** [comment]
I fixed and merged v2.0.0, it should be better now.
Note that v2.0.0 needs node > 4

@Insoleet Could you try for yourself?
Ping @gliide, sorry for the delay :/

**Comment by Insoleet** [comment]
Forgot to tell you, but its fixed ! :+1:
